Modules/gcmodule.c

```diff
@@ -1992,59 +1992,58 @@ static PyMethodDef GcMethods[] = {
     {NULL,      NULL}           /* Sentinel */
 };
 
-static struct PyModuleDef gcmodule = {
-    PyModuleDef_HEAD_INIT,
-    "gc",              /* m_name */
-    gc__doc__,         /* m_doc */
-    -1,                /* m_size */
-    GcMethods,         /* m_methods */
-    NULL,              /* m_reload */
-    NULL,              /* m_traverse */
-    NULL,              /* m_clear */
-    NULL               /* m_free */
-};
-
-PyMODINIT_FUNC
-PyInit_gc(void)
+static int
+gcmodule_exec(PyObject *module)
 {
     GCState *gcstate = get_gc_state();
 
-    PyObject *m = PyModule_Create(&gcmodule);
-
-    if (m == NULL) {
-        return NULL;
-    }
-
+    /* Initialized by _PyGC_Init() early in interpreter lifecycle */
     if (gcstate->garbage == NULL) {
```

Comment:
This test is useless and can be removed. You cannot import the gc module if _PyGC_Init() failed. If _PyGC_Init() fails, Python exits immediately. You can remove the whole test.

I probably forgot to remove it when I wrote _PyGC_Init().